### ISSUE TITLE
Fix subtitle font for UTF-8 compatible font.

### ISSUE THREAD

**Issue by joaoofreitas** [howardchung/watchparty#445]
Hello, 

I've noticed while reading some subtitles in my native language, that the current subtitle font does not support all types of characters.
![image](https://user-images.githubusercontent.com/31630346/162590474-5eb3232c-9937-4e7a-a36f-23c549a3b24d.png)

Should I open a pull request?

Best regards,
João Freitas

**Comment by howardchung** [comment]
I think this is an issue with the original file, not the font? but if you have a PR that fixes, please send it

**Comment by joaoofreitas** [comment]
After investigating the issue, noted that the problem is with the upload of the subtitle blob.

It appears like this after upload, and is not with the encoding of my original file.
 
_After upload:_
![image](https://user-images.githubusercontent.com/31630346/163495319-0bc65f64-3561-4114-91d7-623e4d36cc8b.png)
![image](https://user-images.githubusercontent.com/31630346/163495380-43973d24-8140-4f57-b4a2-0fa135451de8.png)

My file:
![image](https://user-images.githubusercontent.com/31630346/163495480-35481dee-9210-4f27-90c3-5a7bcd518897.png)


Maybe an encoding problem on the SRT to WEBVTT conversion?

Any idea where should I start? @howardchung

Best regards,
João Freitas

**Comment by joaoofreitas** [comment]
I see the problem, is on this package [srt-webvtt](https://www.npmjs.com/package/srt-webvtt).

Maybe we can either contact the developer or patch it temporarily ourselves?

**Comment by howardchung** [comment]
Yeah, if you want to submit a PR to that library or fork it and make a patched version, I can switch to using it.

**Comment by joaoofreitas** [comment]
> Yeah, if you want to submit a PR to that library or fork it and make a patched version, I can switch to using it.

I'll take a look and let you know!

Cheers,
João Freitas

**Comment by joaoofreitas** [comment]
After further research, I noticed that the bug was not in the library but yet in the encoding of my .srt files.

Since no encoding parameter was passed it automatically parses the file as __UTF-8__. 
``` javascript
reader.readAsText(file);
```

Changed this locally and instantly started working.
``` javascript
reader.readAsText(file, "ISO-8859-1");
```
After hard converting my files to __UTF-8__ everything is working.

```bash
iconv --from-code=ISO-8859-1 --to-code=UTF-8 file.srt > file.srt
```

Since it's something I can do in my own, It is not worth to make any changes but may it be worth adding this little detail to the documentation. 
___(I'll open a PR)___

I think we can close this issue.

Thanks,
João Freitas
